src/install/config/mongodb.yaml

```diff
@@ -0,0 +1,110 @@
+agentName: mongodb
+agentType: integration
+title: 'MongoDB monitoring integration'
+metaDescription: 'Install the mongoDB integration'
+introFilePath: 'src/install/mongodb/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
```

Comment:
Nothing super obvious happens when I make the first 3 choices in the Step 1: choose your framework. I'd consider adding a note saying "These choices will determine your installation/configuration code later in this document." This in conjunction with making the guided install instructions more distinct will help clarify what the purpose is between following the docs vs. the guided install.